Mathlib/Deprecated/Aliases.lean

```diff
@@ -1,17 +0,0 @@
-/-
-Copyright (c) 2024 Kim Morrison. All rights reserved.
-Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Kim Morrison
--/
-import Mathlib.Init
-import Batteries.Tactic.Alias
-
-/-!
-Deprecated aliases can be dumped here if they are no longer used in Mathlib,
-to avoid needing their imports if they are otherwise unnecessary.
--/
```

Comment:
Should this file stick around as a catch-all? Given there was only one alias in here before, it seems it is probably not needed, but flagging just in case.

Reply:
I guess it can be recreated whenever needed again

Reply:
@kim-em, since you created this file in #15637, do you have strong opinions?

Reply:
Let's leave it.